saltproc/openmc_deplete.py

```diff
@@ -0,0 +1,75 @@
+import openmc
+import openmc.deplete as od
+import argparse
+
+
+def parse_arguments():
```

Comment:
Why are you allowing the XML files to be in different locations? With normal OpenMC runs, users are expected to have XML files in the same path no?

Reply:
This is just for flexibility. In practice, the user doesn't call this function at all. See the `DepcodeOpenMC.run_depcode` function for an example.

Reply:
It's also an artifact of how I've defined  the `template_inputfiles_path` variable, which has a path for each input file. Normally OpenMC expects the material, geometry, and settings files to be names `materials.xml`, `geometry.xml`, and `settings.xml` respectively. In order for geometry changing to work, we need to be able to have multiple different geomery files in one directory, so they can't all be names `geometry.xml`. Similar, we have an initial material composition that we start the simulation with, but we also have an iteration material file for each depletion step. If both of these had the name `materials.xml`, they could overwrite each other.